libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/NavigationMapRoute.java

```diff
@@ -20,15 +20,17 @@
 import com.mapbox.mapboxsdk.maps.Style;
 import com.mapbox.navigation.base.trip.model.RouteProgress;
 import com.mapbox.navigation.core.MapboxNavigation;
-import com.mapbox.navigation.ui.utils.CompareUtils;
+import com.mapbox.navigation.ui.internal.route.MapRouteLayerProvider;
+import com.mapbox.navigation.ui.internal.route.MapRouteSourceProvider;
+import com.mapbox.navigation.ui.internal.utils.CompareUtils;
 
```

Comment:
Why a bunch of `NavigationMapRouteTest`s are commented out 🤔?

Reply:
Will be taken care separately in a different ticket.